compatibility/versions.bzl

```diff
@@ -22,32 +22,15 @@ sdk_versions = [
     "1.11.0",
     "1.11.1",
     "1.11.2",
-    "1.11.3-snapshot.20211111.6441.0.eed4ad32",
     "1.12.0",
     "1.13.0",
     "1.13.1",
     "1.14.0",
-    "1.14.1-snapshot.20210706.7133.0.8ec16e94",
     "1.14.2",
     "1.15.0",
     "1.16.0",
     "1.17.0",
     "1.17.1",
-    "1.17.2-snapshot.20211124.7857.0.b33b94b8",
-    "1.18.0-snapshot.20210922.7908.0.ced4a272",
-    "1.18.0-snapshot.20210928.7948.0.b4d00317",
-    "1.18.0-snapshot.20211006.8003.0.cfcdc13c",
-    "1.18.0-snapshot.20211013.8071.0.514e8b50",
-    "1.18.0-snapshot.20211019.8113.0.8ff347d8",
-    "1.18.0-snapshot.20211026.8179.0.e474b2d1",
-    "1.18.0-snapshot.20211102.8257.0.7391a3cd",
-    "1.18.0-snapshot.20211109.8328.0.92181161",
-    "1.18.0-snapshot.20211110.8337.0.c5a1f0bb",
-    "1.18.0-snapshot.20211111.8349.0.d938a44c",
-    "1.18.0-snapshot.20211117.8399.0.a05a40ae",
-    "1.18.0-snapshot.20211124.8419.0.39d9afcd",
-    "2.0.0-snapshot.20211123.8463.0.bd2a6852",
-    "2.0.0-snapshot.20211201.8538.0.1d0ff3cc",
     "0.0.0",
```

Comment:
Should we still expect the same kind of problems with this? Is there some change we need to make to avoid running certain tests with the latest `main`?

Reply:
I don’t understand what you mean here. We want to test compatibility of HEAD with older stable versions. If we break it we do need some form of exclusions which we already have.